### PR TITLE
An enhanced PR of PR#3

### PR DESCRIPTION
**Pull request by AbdullahFaqeir**
- Implement same fix from PR #3  and testing it.
- Move table name to configuration.
- Code refactor.
- Fixed default body column value.
- Adopt to php8.

@JhumanJ if you can please add hacktoberfest to this repo's topics.

### CODE REVIEW

**Comment by JhumanJ** [comment]
Hey @AbdullahFaqeir, thanks a lot for your PR! I'll take a look at it now. I'm not sure what you mean by adding hacktoberfest to the repo's topic?

File: src/Http/Controllers/CustomCodeController.php [hunk 1]
```diff
@@ -61,33 +60,28 @@ public function widgetData(Request $request, Tinker $tinker)
...
     {
-        switch ($chartType) {
-            case 'bar_chart':
-                return $this->validateBarChartData($data);
-            case 'line_chart':
-                return $this->validateLineChartData($data);
-        }
-
-        return false;
+        return match ($chartType) {
```
Comment (JhumanJ) [inline src/Http/Controllers/CustomCodeController.php:77]: TIL about match 🙌

File: src/Services/Tinker.php [hunk 2]
```diff
@@ -199,7 +201,7 @@ protected function ignoreCommentsAndPhpTags(array $token)
 
     protected function cleanOutput(string $output): string
     {
-        $output = preg_replace('/(?s)(<aside.*?<\/aside>)|Exit:  Ctrl\+D/ms', '$2', $output);
+        $output = preg_replace('/(?s)(<aside.*?<\/aside>)|Exit: {2}Ctrl\+D/ms', '$2', $output);
```
Comment (JhumanJ) [inline src/Services/Tinker.php:204]: What's up this change? Not sure to understand?

File: database/migrations/create_model-stats_table.php [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
     {
-        Schema::create('model-stats-dashboards', function (Blueprint $table) {
+        Schema::create(Config::get('model-stats.table_name'), function (Blueprint $table) {
             $table->id();
             $table->string('name');
             $table->text('description');
-            $table->jsonb('body')->default('{"widgets":[]}');
+            if (Config::get('database.defaults') === 'pgsql') {
+                $table->jsonb('body')->default('{"widgets":[]}');
+            } else {
```
Comment (JhumanJ) [inline database/migrations/create_model-stats_table.php:18]: Have you had a chance to try this change with an actual mysql DB ?

**Review by JhumanJ — COMMENTED** [review]
I think it's awesome, thanks a lot for your work. Before I merge, I have 2 comments, could you please answer them? @AbdullahFaqeir

File: src/Services/Tinker.php (hunk 2 above)
Comment (AbdullahFaqeir) [reply src/Services/Tinker.php:204]: As the regex is parsing counts of 2, this assures the regex will work more accurately.

File: src/Http/Controllers/CustomCodeController.php (hunk 1 above)
Comment (AbdullahFaqeir) [reply src/Http/Controllers/CustomCodeController.php:77]: hahahah glad I helped with this ^^'

**Comment by JhumanJ** [comment]
Awesome @AbdullahFaqeir ! Code looks good. I'll give it a try on some of my applications and merge the PR by tomorrow. I believe I added the hacktoberfest keywords on the package, but it it's not what you expected, please let me know.
Thanks again!

**Comment by AbdullahFaqeir** [comment]
Great @JhumanJ I'll be waiting for the merge and the new release so I can use it. And yes the keywords are perfect, once you approve and merge the PR it'll be count for me in the fest scoreboard ^^'.

**Pull request merged**